File: 2023-08-25/a/main.cpp

int main(){

  int count = 0;

  while(true){
    count++;
    int n, f;
    std::cin >> n >> f;
    
    if( n==0 || f == 0 )
      break;

    long int value, bill = 0;
    for( int i=0; i<n; i++ )
    {
      std::cin >> value;
      bill += value;
    }
    
    std::cout << "Bill #" << count << " costs " << bill << ": each friend should pay " <<  bill/f << std::endl;

  }
  return 0;
}

File: 2023-08-25/a/main.py
if __name__ ==  '__main__':
    count = 0
    text = ''
    while True:
        count += 1
        bill = 0
        n, f = map( int, input().split() )

        if (n == 0) and (f==0):
            break

        for i in range( n ):
            val = int( input() )
            bill += val

        text += "Bill #{} costs {}: each friend should pay {}".format(count, int(bill), int(bill//f))
        text += '\n\n'

    print(text[:-1])
